.github/workflows/opencode-crossrepo-dispatch.yml

```
@@ -0,0 +1,895 @@
+          # Stage changes (includes new untracked app dir files)
+          git add -A
+          # Ensure staged set is not empty.
+          STAGED=$(git diff --cached --name-only | grep -c . || true)
+          if [ "${STAGED}" = "0" ]; then
+            echo "EMPTY_STAGE BLOCK: zero files staged. Not creating empty branch commit or empty PR."
+            exit 73
+          fi
+          # Commit (must succeed or fail closed. NO || true).
+          git commit -m "feat(${ML}): ${ISSUE_N} OpenCode agent output under app dir
+
+PASAY-OC-DISPATCH-CROSSREPO-001-RETURN1 deterministic publisher.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)